test/gpdb/scripts/test_pitr_gpdb6.sh

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env bash
+set -exo pipefail
+
+PGBACKREST_TEST_DIR=/home/gpadmin/test_pgbackrest
+PGBACKREST_BIN=/usr/local/bin
+GPHOME=/usr/local/greenplum-db-devel
+
+echo "Starting up demo cluster..." > /dev/null
+source $GPHOME/greenplum_path.sh
+pushd gpdb_src/gpAux/gpdemo
+make create-demo-cluster WITH_MIRRORS=true
+source gpdemo-env.sh
+popd
+
+
+echo "Creating backup and log directories for pgbackrest..." > /dev/null
+TEST_NAME=$(basename "${0%.sh}")
```

Comment:
please fix warnings found by https://www.shellcheck.net/
```
$ shellcheck myscript
 
[Line 9:](javascript:setPosition(9, 8))
source $GPHOME/greenplum_path.sh
       ^-- [SC1091](https://www.shellcheck.net/wiki/SC1091) (info): Not following: ./greenplum_path.sh was not specified as input (see shellcheck -x).
 
[Line 12:](javascript:setPosition(12, 8))
source gpdemo-env.sh
       ^-- [SC1091](https://www.shellcheck.net/wiki/SC1091) (info): Not following: gpdemo-env.sh was not specified as input (see shellcheck -x).
 
[Line 92:](javascript:setPosition(92, 25))
-o $PGBACKREST_TEST_DIR/$TEST_NAME/t1_rows_original.out
                        ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([2])), apply [all SC2086](javascript:applyFixCode(2086)))
-o $PGBACKREST_TEST_DIR/"$TEST_NAME"/t1_rows_original.out
 
[Line 106:](javascript:setPosition(106, 18))
    if [[ $(find $segment_backup_dir -maxdepth 1 -type d -name \
                 ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([3])), apply [all SC2086](javascript:applyFixCode(2086)))
    if [[ $(find "$segment_backup_dir" -maxdepth 1 -type d -name \
 
[Line 109:](javascript:setPosition(109, 56))
        echo "Found a backup directory for segment $1: $dirname" > /dev/null
                                                       ^-- [SC2154](https://www.shellcheck.net/wiki/SC2154) (warning): dirname is referenced but not assigned (for output from commands, use "$(dirname ...)" ).
 
[Line 128:](javascript:setPosition(128, 25))
-o $PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_original.out
                        ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([5])), apply [all SC2086](javascript:applyFixCode(2086)))
-o $PGBACKREST_TEST_DIR/"$TEST_NAME"/t2_rows_original.out
 
[Line 154:](javascript:setPosition(154, 8))
rm -rf $MASTER/* $PRIMARY1/* $PRIMARY2/* $PRIMARY3/*
       ^-- [SC2115](https://www.shellcheck.net/wiki/SC2115) (warning): Use "${var:?}" to ensure this never expands to /* .
       ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
                 ^-- [SC2115](https://www.shellcheck.net/wiki/SC2115) (warning): Use "${var:?}" to ensure this never expands to /* .
                 ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
                             ^-- [SC2115](https://www.shellcheck.net/wiki/SC2115) (warning): Use "${var:?}" to ensure this never expands to /* .
                             ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
                                         ^-- [SC2115](https://www.shellcheck.net/wiki/SC2115) (warning): Use "${var:?}" to ensure this never expands to /* .
                                         ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([7,9,11,13])), apply [all SC2086](javascript:applyFixCode(2086)))
rm -rf "$MASTER"/* "$PRIMARY1"/* "$PRIMARY2"/* "$PRIMARY3"/*
 
[Line 155:](javascript:setPosition(155, 8))
rm -rf $MIRROR1/* $MIRROR2/* $MIRROR3/* $DATADIR/standby/*
       ^-- [SC2115](https://www.shellcheck.net/wiki/SC2115) (warning): Use "${var:?}" to ensure this never expands to /* .
       ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
                  ^-- [SC2115](https://www.shellcheck.net/wiki/SC2115) (warning): Use "${var:?}" to ensure this never expands to /* .
                  ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
                             ^-- [SC2115](https://www.shellcheck.net/wiki/SC2115) (warning): Use "${var:?}" to ensure this never expands to /* .
                             ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
                                        ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([15,17,19,20])), apply [all SC2086](javascript:applyFixCode(2086)))
rm -rf "$MIRROR1"/* "$MIRROR2"/* "$MIRROR3"/* "$DATADIR"/standby/*
 
[Line 178:](javascript:setPosition(178, 19))
gpinitstandby -as $HOSTNAME -S $DATADIR/standby/ -P 6001
                  ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.
                               ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([21,22])), apply [all SC2086](javascript:applyFixCode(2086)))
gpinitstandby -as "$HOSTNAME" -S "$DATADIR"/standby/ -P 6001
 
[Line 182:](javascript:setPosition(182, 25))
-o $PGBACKREST_TEST_DIR/$TEST_NAME/t1_rows_restored.out
                        ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([23])), apply [all SC2086](javascript:applyFixCode(2086)))
-o $PGBACKREST_TEST_DIR/"$TEST_NAME"/t1_rows_restored.out
 
[Line 185:](javascript:setPosition(185, 25))
-o $PGBACKREST_TEST_DIR/$TEST_NAME/t2_rows_restored.out
                        ^-- [SC2086](https://www.shellcheck.net/wiki/SC2086) (info): Double quote to prevent globbing and word splitting.

Did you mean: ([apply this](javascript:applyFixIndex([24])), apply [all SC2086](javascript:applyFixCode(2086)))
-o $PGBACKREST_TEST_DIR/"$TEST_NAME"/t2_rows_restored.out

$
```